spec/models/activities/fetcher_integration_spec.rb

```diff
@@ -30,7 +30,7 @@
 
 describe Activities::Fetcher, 'integration' do
   shared_let(:user) { create(:user) }
-  shared_let(:permissions) { %i[view_work_packages view_time_entries view_changesets view_wiki_edits] }
+  shared_let(:permissions) { %i[view_work_packages view_time_entries view_changesets view_wiki_edits view_budgets] }
```

Comment:
@ulferts I have removed all the extra tests and added what I believe you are asking for. Not sure if the` view_budgets` permission needs to also be removed, however.